core/vm/evm.go

```diff
@@ -573,7 +570,8 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 
 	// Reject code starting with 0xEF if EIP-3541 is enabled.
 	if err == nil && len(ret) >= 1 && HasEOFByte(ret) {
-		if evm.chainRules.IsShanghai {
+		if evm.chainRules.IsPrague && isInitcodeEOF {
+			fmt.Printf("FIXME - valid EOF deployment\n")
```

Comment:
Wdym here?

Reply:
leftover debug code - I left it as FIXME to make sure I didn't check it in.  Will clean up.